### ISSUE TITLE
Product / asset uptime tracking 

### ISSUE THREAD

**Issue by ushbot** [ushahidi/platform#534]
Send notification / report when product is down

*Created by @rjmackay on 2014-08-25 15:13:01.*

*Imported from https://phabricator.ushahidi.com/T660*

**Comment by ushbot** [comment]
*Comment by @rjmackay on 2014-08-25 15:14:46:*

This is probably a variation on alerts. If alerts are tied to sets, we just need to be able to create a dynamic set of 'posts' for when a product/entity/asset is "down".

@anarghya can you add any extra info about how this might work?

**Comment by ushbot** [comment]
*Comment by @rjmackay on 2014-08-25 15:24:15:*

I think T661 is related.. we probably can't trigger alerts for an asset/product unless we can map it first.

**Comment by ushbot** [comment]
*Comment by anarghya on 2014-08-26 02:46:29:*

Yes, seems like this is very much related to alerts. If the product stops working, they want a message pushed to the server that says "the power plant went down" or something like that. So they would like it to be basically instant (if possible) or with as little delay as possible. This requires work from the partners' end because their hardware has to have some kind of sensor or something that tells us that the product is down, right?

**Comment by ushbot** [comment]
*Comment by @rjmackay on 2014-08-26 10:43:39:*

Yes they need to have sensors. I'm flagging T8 for sensor integration..
"they want a message pushed to the server that says "the power plant went down" " .. pushed to the server? or sent out via SMS/email?
I'd assume once T8 is done we should have regular messages from sensors with current status (up/down/voltage/whatever).. that just becomes another report in the system.

**Comment by ushbot** [comment]
*Comment by @rjmackay on 2015-02-26 14:03:48:*

@anarghya Any more info / updates on this?

**Comment by ushbot** [comment]
*Comment by @rjmackay on 2015-02-26 14:13:06:*

If T8 is no longer relevant for Google.. does that mean this is out too?
